.travis.yml

```diff
@@ -49,9 +52,16 @@ matrix:
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm-nightly
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm-nightly
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
+
+addons:
+  postgresql: '9.4'
```

Comment:
Its from the travis-ci devblog- 9.4 support is experimental, and this is
how you enable it apparently.
On Mar 31, 2015 3:16 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/doctrine/dbal/pull/825#discussion_r27457085>:
>
> >      - php: hhvm-nightly
> >        env: DB=pgsql POSTGRESQL_VERSION=9.1 # driver currently unsupported by HHVM
> >      - php: hhvm-nightly
> >        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
> >      - php: hhvm-nightly
> >        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
> > +    - php: hhvm-nightly
> > +      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
> > +
> > +addons:
> > +  postgresql: '9.4'
>
> Never saw this: what's that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/dbal/pull/825/files#r27457085>.
>